.github/workflows/python-build.yaml

```
@@ -15,8 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8]
-        poetry-version: [1.1.6]
+        python-version: [3.10]
```

This needs to have quotes otherwise github treats this as 3.1...
Refernce: https://github.com/Nike-Inc/knockoff-factory/blob/master/.github/workflows/python-build.yaml#L18
```suggestion
        python-version: ["3.10"]
```

---

good call! changed in f73ff94a733a50d1e9a6b0a87f5bbdf3705767fd